### PR TITLE
Add SwiftLint action file

### PR DESCRIPTION
**Pull request by terlan98**
Note: It's okay for the SwiftLint action to fail for now. It will succeed once #151 is merged

### CODE REVIEW

File: .github/workflows/swiftlint.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  SwiftLint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: GitHub Action for SwiftLint
+        uses: norio-nomura/action-swiftlint@3.2.1
+      - name: GitHub Action for SwiftLint with --strict
+        uses: norio-nomura/action-swiftlint@3.2.1
+        with:
+          args: --config swiftlint.yml --strict
```
Comment (maximiliansoelch) [inline .github/workflows/swiftlint.yml]: We don't need four SwiftLint runs, just the one for changed files and our custom SwiftLint config file
```suggestion
```

File: .github/workflows/swiftlint.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        with:
+          args: --config swiftlint.yml
+        env:
+          DIFF_BASE: ${{ github.base_ref }}
+      - name: GitHub Action for SwiftLint (Different working directory)
+        uses: norio-nomura/action-swiftlint@3.2.1
+        with: 
+          args: --config swiftlint.yml
+        env:
+          WORKING_DIRECTORY: Source
```
Comment (maximiliansoelch) [inline .github/workflows/swiftlint.yml]:
```suggestion
```

**Comment by maximiliansoelch** [comment]
The gh action from norio-nomura is quite old. Can we use this fork instead: https://github.com/sinoru/actions-swiftlint

File: .github/workflows/swiftlint.yml
Comment (maximiliansoelch) [inline .github/workflows/swiftlint.yml:1]: Please also change to the other gh action mentioned in my comment above

**Review by maximiliansoelch — CHANGES_REQUESTED** [review]

File: .github/workflows/swiftlint.yml
Comment (terlan98) [reply .github/workflows/swiftlint.yml:1]: The other gh action failed for some reason. I will try it again

**Review by maximiliansoelch — APPROVED** [review]

**Pull request merged**